modern/src/settings/MaintenancePage.js

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
+import t from '../common/localization';
+import EditItemView from '../EditItemView';
+import { Accordion, AccordionSummary, AccordionDetails, makeStyles, Typography, TextField, FormControl, InputLabel, MenuItem, Select, } from '@material-ui/core';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import EditAttributesView from '../attributes/EditAttributesView';
+import positionAttributes from '../attributes/positionAttributes';
+import { useAttributePreference } from '../common/preferences';
+import { distanceConverter, speedConverter } from '../common/converter';
+
+const useStyles = makeStyles(() => ({
+  details: {
+    flexDirection: 'column',
+  },
+}));
+
+const MaintenancePage = () => {
+  
+  const classes = useStyles();
+  const [item, setItem] = useState();
+  const [labels, setLabels] = useState({start: '', period: ''});
+
+  const speedUnit = useAttributePreference('speedUnit');
+  const distanceUnit = useAttributePreference('distanceUnit');
+
+  const options = [];
+
+  Object.entries(positionAttributes).map(([key, value]) => {
+    if (value.type === 'number') {
+      options.push({key, name: value.name, type: value.type})
+    }
+  });
+
+  const handleChange = event => {
```

Comment:
We should probably rename it to be more clear.

Reply:
done